cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/AbstractParameters.java

```diff
@@ -31,12 +32,17 @@ public AbstractParameters(HttpServletRequest request) {
   }
 
   protected void initParameters() throws UnsupportedEncodingException {
+    _initialized = true;
     _endPoint = ParameterUtils.endPoint(_request);
     _json = ParameterUtils.wantJSON(_request);
   }
 
   @Override
   public boolean parseParameters(HttpServletResponse response) {
+    if (_initialized) {
```

Comment:
Can we update the JavaDoc of `CruiseControlParameters#parseParameters` to indicate that if it is initialized, it returns `false`?